sound/soc/intel/boards/Kconfig

```diff
@@ -31,15 +31,29 @@ config SND_SOC_INTEL_HDA_DSP_COMMON
 
 config SND_SOC_INTEL_SOF_MAXIM_COMMON
 	tristate
+	select SND_SOC_MAX98357A
+	select SND_SOC_MAX98373_I2C
+	select SND_SOC_MAX98390
 
 config SND_SOC_INTEL_SOF_REALTEK_COMMON
 	tristate
+	select SND_SOC_RT1011
+	select SND_SOC_RT1015
+	select SND_SOC_RT1015P
+	select SND_SOC_RT1308
 
 config SND_SOC_INTEL_SOF_CIRRUS_COMMON
 	tristate
+	select SND_SOC_CS35L41_I2C
 
 config SND_SOC_INTEL_SOF_SSP_COMMON
 	tristate
+	select SND_SOC_DMIC
+	select SND_SOC_HDAC_HDMI
+	select SND_SOC_INTEL_HDA_DSP_COMMON
+	select SND_SOC_INTEL_SOF_CIRRUS_COMMON
+	select SND_SOC_INTEL_SOF_MAXIM_COMMON
+	select SND_SOC_INTEL_SOF_REALTEK_COMMON
```

Comment:
yeah, that seems wrong. SSP should never have to select HDAC_HDMI or DMIC. I am not in agreement here.